src/loaders/workspace_uploader/workspace_uploader.py

```diff
@@ -200,44 +204,44 @@ def _get_parser():
     return parser
 
 
-def _get_yaml_file_path(assembly_dir: str) -> str:
+def _get_yaml_file_path(obj_dir: str) -> str:
     """
     Get the uploaded.yaml file path from collections source directory.
     """
-    file_path = os.path.join(assembly_dir, _UPLOADED_YAML)
+    file_path = os.path.join(obj_dir, _UPLOADED_YAML)
     Path(file_path).touch(exist_ok=True)
     return file_path
 
 
-def _get_source_file(assembly_dir: str, assembly_file: str) -> str:
+def _get_source_file(obj_dir: str, obj_file: str) -> str:
     """
-    Get the sourcedata file path from the assembly directory.
+    Get the sourcedata file path from the WS object directory.
     """
-    if not os.path.islink(assembly_dir):
-        raise ValueError(f"{assembly_dir} is not a symlink")
-    src_file = os.path.join(os.readlink(assembly_dir), assembly_file)
+    if not os.path.islink(obj_dir):
+        raise ValueError(f"{obj_dir} is not a symlink")
+    src_file = os.path.join(os.readlink(obj_dir), obj_file)
     return src_file
 
 
 def _upload_assemblies_to_workspace(
     asu: AssemblyUtil,
     workspace_id: int,
     load_id: int,
-    assembly_tuples: list[_AssemblyTuple],
+    ws_obj_tuples: list[_WSObjTuple],
 ) -> tuple[str, ...]:
     """
     Upload assembly files to the target workspace in batch. The bulk method fails
     and an error will be thrown if any of the assembly files in batch fails to upload.
     Returns the list of workspace UPAs for the created objects in the same order as
-    `assembly_tuples`.
+    `ws_obj_tuples`.
     """
     inputs = [
         {
-            "file": assembly_tuple.container_internal_assembly_path,
-            "assembly_name": assembly_tuple.assembly_name,
+            "file": getattr(obj_tuple, _CONTAINER_OBJ_PATH),
+            "assembly_name": getattr(obj_tuple, _OBJ_NAME),
```

Comment:
The point of making a named tuple is so you can use dot syntax to access its members - why are we using getattr?

Reply:
in case we need to change member names, I don't need to go through the whole file again.

Reply:
A refactoring tool should be able to do that for you in one shot

Reply:
👍